crates/iroha_data_model/src/parameter.rs

```diff
@@ -302,7 +298,6 @@ mod model {
         IntoSchema,
     )]
     #[ffi_type(opaque)]
-    #[serde(rename_all = "snake_case")]
     pub enum Parameter {
```

Comment:
FYI @BAStos525 @RamilMus

- To set parameters without updating the image, run `cat parameter.json | iroha parameter set`, where [this](https://github.com/hyperledger-iroha/iroha/blob/f348b9a8a46123b4c5ff1bdda9bccf5103933a5a/crates/iroha_cli/samples/parameter.json) is an example of `parameter.json`.
- To set parameters after updating the image, [this](https://github.com/s8sato/iroha/blob/feat/complete_cli-backport-testnet-2.0.0-rc.1-fixup/crates/iroha_cli/samples/parameter.json) is an example of `parameter.json`.